From 92148a4be791614eaf4582ea540cff82e27134de Mon Sep 17 00:00:00 2001
From: Ondrej Holy <oholy@redhat.com>
Date: Tue, 11 Jun 2013 12:58:28 +0000
Subject: mouse: Rename "Content sticks to fingers"

...to "Natural scrolling". The previous name was a little bit
disgusting, and people didn't understand its purpose. Using the
same name as OSX means that people will either know it, or be less
afraid of testing it.

https://bugzilla.gnome.org/show_bug.cgi?id=689128
---
diff --git a/panels/mouse/gnome-mouse-properties.ui b/panels/mouse/gnome-mouse-properties.ui
index cd77080..3735903 100644
--- a/panels/mouse/gnome-mouse-properties.ui
+++ b/panels/mouse/gnome-mouse-properties.ui
@@ -655,7 +655,7 @@
                                 </child>
                                 <child>
                                   <object class="GtkCheckButton" id="natural_scroll_toggle">
-                                    <property name="label" translatable="yes">C_ontent sticks to fingers</property>
+                                    <property name="label" translatable="yes" comments="Translators: This switch reverses the scrolling direction for touchpads. The term used comes from OS X so use the same translation if possible. ">_Natural scrolling</property>
                                     <property name="use_action_appearance">False</property>
                                     <property name="visible">True</property>
                                     <property name="can_focus">True</property>

